src/hooks/pageHooks/useCreatePageHook.jsx

```diff
@@ -53,10 +53,23 @@ export function useCreatePageHook(params, queryParams) {
         )
         if (queryParams && queryParams.onSuccess) queryParams.onSuccess()
       },
-      onError: () => {
-        errorToast({
-          description: `A new page could not be created. ${DEFAULT_RETRY_MSG}`,
-        })
+      onError: (err) => {
+        // check for new err format
+        if (
+          err.response &&
```

Comment:
hmm this condition seems like something that we are going to end up using often, should we export a 
`isV2Error()` function to be used by multiple parts of our codebase instead? 
concretely, 
```
export function isV2Error(err: unknown) {
if (err.response &&
     err.response.data &&
     err.response.data.error &&
     err.response.data.error.isV2Err) return true 
return false
}
```

Reply:
actually we already have a utility for this called `getAxiosErrorMessage` - could we use that instead?

Reply:
good point, fixed!